scene/resources/mesh.cpp

```diff
@@ -1021,7 +1021,7 @@ void _fix_array_compatibility(const Vector<uint8_t> &p_src, uint32_t p_old_forma
 							dst[0] = (uint16_t)CLAMP(res.x * 65535, 0, 65535);
 							dst[1] = (uint16_t)CLAMP(res.y * 65535, 0, 65535);
 						}
-						src_offset += sizeof(uint16_t) * 2;
+						src_offset += sizeof(float) * 3;
```

Comment:
This should be four floats so that it matches the size here:
https://github.com/godotengine/godot/blob/11c14ce9515483ce2731cc259819b2b4a49450dc/servers/visual_server.cpp#L942

Reply:
dang it, fixed.